### DISCUSSION TITLE
Legal address of RustDesk

### DISCUSSION THREAD

**Discussion by alexander-krug** [rustdesk/rustdesk#9695] [Q&A] [answered]
Hi there,

the Website [rustdesk.com](https://rustdesk.com/team/) lists Purslane Ltd. as the company name, doing research online led me to this official UK government entry: 
https://find-and-update.company-information.service.gov.uk/company/09220299/filing-history
According to this, Purslane Ltd. stopped existing in June 2023. 

But the terms on their website still reference Purslane Ltd.: [rustdesk.com](https://rustdesk.com/terms.html)

The only other information on their website, specifically this [rustdesk.com](https://rustdesk.com/team/), states the location of RustDesk as "18A, Canberra Drive, Yishun, Singapore".

So what is the legal address of RustDesk now?

**Comment by xlionjuan** [comment]
It is the one in SG, not the one in UK.

**Reply by RickkeeC** [reply]
6 hours ago "The sale information is very old"  Then Oct21, It is the one in SG.   Really? Humm, Sketchy as F.

**Comment by RickkeeC** [comment]
Sketchy as F.  It traces to an apartment that is FOR RENT. I researched in September ''24 from icann lookup that has now been cloaked with  Google "privacy protection" that only has abuse@godaddy.com information.  MX Record goes to ZoHo in China.    At least HopToDesk was able to pass the Windows Store vetting process. Idk if that means much considering how many rogue apps make it into the G00gle PlayStore. Rustdesk claims to be "open source" except some of the server code is encrypted and cannot be accessed, negating the important protection of total transparency.  I'd like to deploy it across our 1,000 endpoints but scared as F. because of this and cannot proceed with the paid version starting at usd 600/year maybe more for all these 'puters. + this is their github and they are not responding to inquiries about transparency.   To sumarize, "Too Sketchy, God only knows what kinds of back-doors could be built into that encrypted code.

We are watching HoptoDesk as a strong 2nd candidate, but it is missing some important features in RustDesk server and client. Looks like they ditched 64-bit app for Windows Store as only 32-bit exe is found now.  * Major advantage is that it includes a beta "console" that works as a pretty decent phone book.  Now if I could only reboot the the clients server and connect with SSH to run Windows Remote Desktop.

![rustdesk-whois-physical-address](https://github.com/user-attachments/assets/d3ecd734-5378-4742-9019-5e2c42c4e52b)
![RustDesk-Room4Rent](https://github.com/user-attachments/assets/50d0a274-232d-4615-b9ec-1e1eef3435eb)

**Reply by Poordeveloper** [reply]
So you trust a copycat of RustDesk rather than the original open source version, just because its price is higher than your expectation, super selfish and awful guys. No body force you use the paid pro version, you can use the open source free self-hosting version. I guess you will say gods know there is backdoor in the open source free self-hosting version, because I do not know the code, nobody can convince guys like you only yourself who only live in your own world.

https://github.com/rustdesk/rustdesk/discussions/2778
https://github.com/rustdesk/rustdesk/discussions/1590

**Reply by rustdesk** [reply, marked as answer]
I bought this property in 2014/2015, the sale you see on propertyguru is very old. You should check RustDesk.exe EV signature, the address is verfied and vetted by SSL issuer, they are more trustworthy than MS store.

**Reply by RickkeeC** [reply]
Thank you for the reply.

I don’t trust anybody without verification, not even my Monsignor.  However, I perceive trust level for Hop2 just a tiny-wee bit more.

How do you justify calling the developers “copycats”.  Is not that what a fork is for? 
They do state, originally forked from RustDesk in help/about.  I don’t know much about AGPL 3.0, both use the same.
IMHO the only open-source part of RustDesk is that you take on volunteer contributors that work hard for free, and import it into 
your proprietary, encrypted code base and charge money for it. 

Maybe a new term is needed to better define.  How about Copyright-Open-Source AGPL 3.0 Or Open-Source-Kinda AGPL 3.0.
The community license could be “honest” Open-Source like Apache and BSD. Your Closed-Source product license can then be modeled 
after the Bill Gates license and just “rent it” so you can so you can take it away.  Alas, Open-Source, Copy-Left and Permissive
is the subject for another reddit.   If I can’t see it and modify it, it Ain’t Open-Source.

![image](https://github.com/user-attachments/assets/71f8e0f0-3033-438f-8d63-c4b4ddbc5622)

2.)	Hop2 made it to the Windows Store, EV Signed, (+1), Hop2 MSIX Store app is signed too. (+1)  RustDesk EV Code sign (+1).  RustDesk yet another lie about Hop2 (-1).  Total score on Certs  Hop2 (2) RustDesk (0) 

![image](https://github.com/user-attachments/assets/eb7b70d0-06ee-422c-b160-623db5c0f7a8)

3.)	Hop2 is providing a beta console for which is fancy word for phone book. (+1)
4.)	RustDesk $600 starting price for up to 300 assets.  We have 1,000+ and growing    That’s crazy money for a phone book. ( -1)
5.)	Free version is totally unorganized, too crippled.  No sort order, have to hand-pecker in every connection and connect (-1)
on every device to save it, on one computer.  
Must maintain manual database of RustDesk connectiods just to be safe. Yes, I can copy/past from appdata land 
and can retrieve connectiods with PowerShell from individual machines (Thank you for that) (+1) 
but unusable with multiple Engineers. 

Total Scores
Hop2 (+3) RustDesk (+1)

No score given brand name impression by casual end users.  HopToDesk, groovy.  RustDesk?  WTF why am I installing something from a rusty old desktop on my shiny new PC.  The RD name only means something to us propeller heads.

**Reply by RickkeeC** [reply]
By the way, how long ago did you sell it and where did you move to?  No links have been updated, not even here in about, address is obscured by domain privacy protection.  My apology for posting the link to the $1,000 a month flat.  The unit number is actually a $4,000 condo for rent.  Sketchier as F. more than before.  (-1) but it is a really good app, don't get me wrong about that. (+1).  Keep up the good work, and work on this "transparency-trust" issue.  Marked as Answered (-1) Come on....

**Reply by RickkeeC** [reply]
There you go with the personal attacks again.  "Guys like you...." It's not just the money, that is just part of the score.  You are missing the point entirely. I don't know enough about either company, just offering my opinion and comparison based on stated facts with _verifiable receipts_.  You didn't answer ANY QUESTIONS, just sidestep and insult.  I've heard enough of that for the election and now have I to put up with it for another 4 years. You can only reply to the issues with personal insults?  Grow up. I thought personal attacks are against the rules in /r. You must be really sore about their fork.  Answer at least one question please.   From the rustdesk.com home page:    THEN we can HONESTLY mark it "ANSWERED"
![image](https://github.com/user-attachments/assets/4f02040b-2f91-466f-912c-cd0fb6999c1d)

**Reply by RickkeeC** [reply]
We will have to wait and see how you do with the GSA candidite thing.  That would be a +10
Unless you are unwilling to give them the real company address.
![image](https://github.com/user-attachments/assets/a4897d02-78de-499d-82ec-32319c63b196)

I even gave a few compliments on the app, and you must have missed those too. Over-priced? That is a state of mind for some.  If I purchase anything to get out of$4500 / year Splashtop subscription it would be JumpDeskTop.  Rust is a close second, except for this trust hangup I have.  Am I the only one?



.
